src/Compilers/Core/MSBuildTaskTests/CscTests.cs

```diff
@@ -334,21 +334,30 @@ public void RefOnly()
         }
 
         [Fact]
-        public void NullableReferenceTypes_True()
+        public void NullableReferenceTypes_Enabled()
         {
             var csc = new Csc();
             csc.Sources = MSBuildUtil.CreateTaskItems("test.cs");
-            csc.NullableReferenceTypes = true;
-            Assert.Equal("/nullable+ /out:test.exe test.cs", csc.GenerateResponseFileContents());
```

Comment:
Since `nullable+` and `nullable-` are still valid options, don't we want to keep these tests as well?